assets/src/blocks/Splittwocolumns/SplittwocolumnsEditor.js

```diff
@@ -0,0 +1,75 @@
+import { Fragment } from '@wordpress/element';
+import { BLOCK_NAME, VERSION } from './SplittwocolumnsBlock';
+import { SplittwocolumnsFrontend as Frontend } from './SplittwocolumnsFrontend';
+import { SplittwocolumnsSettings as SidebarSettings } from './SplittwocolumnsSettings';
+import { SplittwocolumnsInPlaceEdit as InPlaceEdit } from './SplittwocolumnsInPlaceEdit';
+
+const { apiFetch } = wp;
+const { useSelect } = wp.data;
+
+
+const useImage = (image_id, current_url) => {
+  const { image = {url: current_url} } = useSelect(select => {
+    return image_id ? { image: select('core').getMedia(image_id) } : { image: null };
```

Comment:
Maybe here you could write the check inside of the object, to avoid a bit of duplicate code? Sthg like `{ image: image_id ? select('core').getMedia(image_id)  : null }`